Core/Core/Assignments/AssignmentDetails/AssignmentDetailsView.swift

```diff
@@ -41,27 +43,30 @@ public struct AssignmentDetailsView: View, ScreenViewTrackable {
         screenViewTrackingParameters = ScreenViewTrackingParameters(
             eventName: "/courses/\(courseID)/assignments/\(assignmentID)"
         )
-
+        isTeacherEnrollment = course.first?.enrollments?.contains(where: { $0.isTeacher || $0.isTA }) == true
```

Comment:
Just to double check: do we receive non-active enrollments here? In case yes we should check for the active status as well. (Think about a user who was accidentally added to a course a teacher and then got removed and added back again as student, in this case there might be two enrollments: one inactive teacher and one active student.) I know that this code was like this before the PR so only do it if it's easy.

Reply:
I added an active state check as well.